generators/client/templates/angular/package.json.ejs

```diff
@@ -162,11 +162,13 @@
       "git add"
     ]
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
   <%_ } _%>
   "scripts": {
-    <%_ if(!skipCommitHook) { _%>
```

Comment:
Please retain the condition in the new husky block as well

Reply:
It is the case, the husky blocked is merged with the lint-staged one (line 158).